backend/src/main/java/com/java/academy/gym/model/UserProfile.java

```diff
@@ -0,0 +1,60 @@
+package com.java.academy.gym.model;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Table;
+import java.util.Objects;
+
+@Entity
+@Table(name = "USER_PROFILES")
+public class UserProfile extends BaseEntity {
```

Comment:
how are you going to get this information?

Reply:
Which information?